src/test/java/teammates/test/driver/TestStringHelper.java

```diff
@@ -0,0 +1,23 @@
+package teammates.test.driver;
+
+import teammates.common.util.StringHelper;
+
+/**
+ * Holds additional methods for {@link StringHelper} used only in tests.
+ */
+public final class TestStringHelper {
```

Comment:
Why i used test as prefix was to make it more self explanatory but since as we have not used it for other classes so we can drop the prifix.
So what should we do?